test/e2e_test/testbench/testbench.go

```diff
@@ -198,6 +198,7 @@ func SetupTests(t *testing.T) (shieldv1beta1.ShieldServiceClient, *config.Shield
 			GRPC: server.GRPCConfig{
 				Port: apiGRPCPort,
 			},
+			DefaultSystemEmail:  OrgAdminEmail,
```

Comment:
let's define a new email var for system? this will avoid any confusions for existing and new tests

Reply:
I was suggesting to have another variable for DefaultSystemEmail. The idea to not use OrgAdminEmail is because it might be used in some other test cases and might bring in an assumption that Org Admin email and System Email needs to be the same (which is not the case, CMIIW?).

Reply:
with current logic in the service, user that is not registered cannot create user. we can do this but at first OrgAdminEmail must be registered by the DefaultSystemEmail first. is it ok to make changes this way?

Reply:
Or I will make changes so the API test will use OrgAdminEmail but BootstrapUser() would use DefaultSystemEmail.

Reply:
added changes so admin email created in the setup test process and any other process(bootstraping and test) are using admin email.

https://github.com/goto/shield/pull/37/commits/eeaaad7eb22d557a348fb6ad13e306fc65dc9068#diff-6a0e262d876f92617d33d2fe7bdfb1c0be2835a5837febb8074ce3a38a40bc26R232-R263

Reply:
LGTM